applications/directory/src/components/LogIn.js

```diff
@@ -0,0 +1,168 @@
+import { useState } from 'react';
+import Alert from '@mui/material/Alert';
+import * as Yup from 'yup';
+import { useFormik } from 'formik';
+import Card from '@mui/material/Card';
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import InputAdornment from '@mui/material/InputAdornment';
+import MailOutlineIcon from '@mui/icons-material/MailOutline';
+import Button from '@mui/material/Button';
+import VerifiedIcon from '@mui/icons-material/Verified';
+import { useEmailValidation, useAuth } from '../hooks';
+
+const LogIn = () => {
+  const [ifValidEmail, setIfValidEmail] = useState('');
+  const { ifEmailExists } = useEmailValidation();
+  const { authError, onSubmit: onSubmitCredentials } = useAuth();
+  // formik
+  const {
+    handleBlur,
+    handleChange,
+    handleSubmit,
+    setFieldValue,
+    setValues,
+    setTouched,
+    resetForm,
+    values,
+    isValid,
+    isSubmitting,
+    dirty,
+    errors,
+    touched
+  } = useFormik({
+    initialValues: {
+      email: '',
+      password: ''
+    },
+    // enableReinitialize: true, // lets the form to go back to initial values if reset form
+    validationSchema: Yup.object().shape({
+      email: Yup.string()
+        .email('Please provide a valid email address.')
+        .test('email', ifValidEmail, () => ifValidEmail === '')
+        .required('Please enter your email address.')
+    }),
+    onSubmit: async ({ email, password }) =>
+      onSubmitCredentials(email, password)
+  });
+  return (
+    <Card
+      sx={{
+        flex: 1,
+        textAlign: 'center',
+        border: '2px solid',
+        borderColor: 'secondary.secondary',
+        '& p.create': {
```

Comment:
You know my opinion on too much nesting like this ;)

Reply:
There was not any job done on styling on this component. It will be done in responsive-styling branch or maybe by the new developer joining our team :)